cmux_test.go

```diff
@@ -181,6 +182,77 @@ func runTestRPCClient(t *testing.T, addr net.Addr) {
 	}
 }
 
+func TestTimeout(t *testing.T) {
+	defer leakCheck(t)()
+	lis, Close := testListener(t)
+	//defer Close()
+	result := make(chan int, 5)
+	testDuration := time.Second
+	m := New(lis)
+	m.SetReadTimeout(testDuration)
+	http1 := m.Match(HTTP1Fast())
+	any := m.Match(Any())
+	go func() {
+		_ = m.Serve()
+	}()
+	go func() {
+		con, err := http1.Accept()
+		if err != nil {
+			result <- 1
```

Comment:
This is a large test and is kinda difficult to read. Instead of using, numbers can you please create constants with readable/self-documenting info?

1 should be http1Error, or 3 should be anyError